static/app/components/events/interfaces/debugMeta/debugImageDetails/index.tsx

```diff
@@ -70,16 +70,30 @@ export class DebugImageDetails extends AsyncComponent<Props, State> {
 
     const {debug_id, candidates = []} = image;
 
-    const uploadedDebugFiles = this.getUplodedDebugFiles(candidates);
+    const hasUploadedDebugFiles = this.getUploadedDebugFiles(candidates);
     const endpoints: ReturnType<AsyncComponent['getEndpoints']> = [];
 
-    if (uploadedDebugFiles) {
+    if (hasUploadedDebugFiles) {
       endpoints.push([
         'debugFiles',
         `/projects/${organization.slug}/${projSlug}/files/dsyms/?debug_id=${debug_id}`,
         {
           query: {
-            file_formats: ['breakpad', 'macho', 'elf', 'pe', 'pdb', 'sourcebundle'],
+            // FIXME(swatinem): Ideally we should not filter here at all,
+            // though Symbolicator does not currently report `bcsymbolmap` and `il2cpp`
+            // candidates, and we would thus show bogus "unapplied" entries for those,
+            // which would probably confuse people more than not seeing successfully
+            // fetched candidates for those two types of files.
```

Comment:
A workaround for this would be to filter these candidates (on the symbolicator side) to only those from the internal sentry source.